packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -236,10 +236,28 @@ class ScrollableBase extends Component {
 		};
 
 		configureSpotlightContainer(props);
+
+		this.state = {
+			prevSpotlightId: props['data-spotlight-id'],
+			prevFocusableScrollbar: props.focusableScrollbar
+		};
 	}
 
-	UNSAFE_componentWillReceiveProps (nextProps) {
-		configureSpotlightContainer(nextProps);
+	static getDerivedStateFromProps (props, state) {
+		const
+			{'data-spotlight-id': spotlightId, focusableScrollbar} = props,
+			{prevSpotlightId, prevFocusableScrollbar} = state;
+
+		if (prevSpotlightId !== spotlightId || prevFocusableScrollbar !== focusableScrollbar) {
+			configureSpotlightContainer(props);
```

Comment:
We really, really shouldn't be doing side effects within this static method.  Also, there should be a better way to deal with this other than adding state just for the sole purpose of having it available within `getDerivedStateFromProps`.